framework/src/transfers/MultiAppNearestNodeTransfer.C

```diff
@@ -22,6 +22,10 @@
 #include "libmesh/parallel_algebra.h"
 #include "libmesh/dof_object.h"
 
+// TIMPI includes
+#include "timpi/communicator.h"
```

Comment:
All MooseObjects are ParallelObjects - you shouldn't ever need to include the comm includes